systematic-and-applied-microbiology.csl

```diff
@@ -112,14 +112,14 @@
     </choose>
   </macro>
   <citation collapse="citation-number">
-    <sort>
-      <key variable="citation-number" sort="ascending"/>
-    </sort>
     <layout delimiter="," prefix="[" suffix="]">
```

Comment:
OK Sebastian, I understood. Please ignore this pull request and I will create another pull request after fixing this. One author is waiting to use this style sheet.

From: Sebastian Karcher [mailto:notifications@github.com]
Sent: Wednesday, March 25, 2015 7:33 PM
To: citation-style-language/styles
Cc: Murugesan, Sankara Narayanan (ELS-CHN)
Subject: Re: [styles] Update systematic-and-applied-microbiology.csl (#1474)


In systematic-and-applied-microbiology.csl<https://github.com/citation-style-language/styles/pull/1474#discussion_r27122464>:

> @@ -112,14 +112,14 @@

>      </choose>

>    </macro>

>    <citation collapse="citation-number">

> -    <sort>

> -      <key variable="citation-number" sort="ascending"/>

> -    </sort>

>      <layout delimiter="," prefix="[" suffix="]">

I don't think that's right. With this deleted, you could get citations out of order in a single citation in the text, things like, [14, 3, 7, 2]. That can't be right. You can leave this in and still get alphabetical sorting in the bibliography.

—
Reply to this email directly or view it on GitHub<https://github.com/citation-style-language/styles/pull/1474/files#r27122464>.